packages/devtools_app/web/index.html

```diff
@@ -52,6 +52,22 @@
       });
     }
   </script>
+
+  <!-- Script to fail gracefully when CanvasKit crashes. -->
+  <script>
+    var realWarn = window.console.warn.bind(console);
+    window.console.warn = function(obj) {
+      if( typeof(obj) == "string" && obj.indexOf("CanvasKit") != -1 && obj.indexOf("font") != -1) { 
+        window.alert(
+          "The Flutter CanvasKit Skia Backened has encountered a fatal error and needs to restart. " +
```

Comment:
maybe: 

"Oh no! Fatal error in the Flutter CanvasKit backend. We are aware of this issue and working hard on a fix. Press OK to restart DevTools."

Reply:
That sounds good as well. I pasted in Devon's suggestion first. Feel free to switch to this suggestion before committing.